addons/controllers/azurediskcsi/azurediskconfig_utils.go

```diff
@@ -0,0 +1,78 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	clusterapiv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/vmware-tanzu/tanzu-framework/addons/pkg/constants"
+	csiv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/addonconfigs/csi/v1alpha1"
+)
+
+const (
+	defaultDataValueNameSpace          = "tkg-system"
```

Comment:
kube-system

Reply:
Thanks, will add a fix patch on the new branch